UnityProject/Assets/Scripts/Input System/InteractionV2/InteractionUtils.cs

```diff
@@ -14,31 +14,60 @@ public static class InteractionUtils
 	/// Use this on client side to request an interaction. This can be used to trigger
 	/// interactions manually outside of the normal interaction logic.
 	/// Request the server to perform the indicated interaction using the specified
-	/// component. The server will still validate the interaction and only perform
+	/// component (or let the server determine the component if interactableComponent is null). The server will still validate the interaction and only perform
 	/// it if it validates.
 	///
 	/// Note that if interactableComponent implements IClientSideInteractable for this interaction type,
 	/// nothing will be done.
 	/// </summary>
 	/// <param name="interaction">interaction to perform</param>
-	/// <param name="interactableComponent">component to handle the interaction</param>
+	/// <param name="interactableComponent">component to handle the interaction, null
+	/// if the server should determine which component will trigger from the interaction</param>
 	/// <typeparam name="T"></typeparam>
-	public static void RequestInteract<T>(T interaction, IBaseInteractable<T> interactableComponent)
+	public static void RequestInteract<T>(T interaction, IBaseInteractable<T> interactableComponent = null)
 		where T : Interaction
 	{
 		if (!Cooldowns.TryStartClient(interaction, CommonCooldowns.Instance.Interaction)) return;
 		RequestInteractMessage.Send(interaction, interactableComponent);
 	}
 
+
+	/// <summary>
+	/// Checks if this component would trigger and sends the interaction request to the server if it does. Doesn't
+	/// send a message if it only triggered an IClientInteractable (client side only) interaction.
+	/// </summary>
+	/// <param name="interactable"></param>
```

Comment:
The parameter documentation is empty, I think you should remove it or complete it

Reply:
@daalta fixed